AbilityChanger/Abilities/AcidArmor.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
:suspect:

Reply:
I didn't add neither Ismas or lantern zoteZote

Reply:
that's fair but we can yeet the files till we do ig?